fggs/multi.py

```diff
@@ -189,6 +236,7 @@ def multi_solve(a: MultiTensor, b: MultiTensor, transpose: bool = False) -> Mult
     # Make copies of a and b flattened into matrices and vectors, respectively
     shapes = a.shapes[0]
     # assert a.shapes[0] == a.shapes[1] == b.shapes[0]
+
```

Comment:
Should there be a call to `_order_nonterminals`?

Reply:
oops! accidentally took this out